release-calendar/src/client/components/Calendar.tsx

```diff
@@ -23,10 +23,16 @@ import timeGridPlugin from '@fullcalendar/timegrid';
 
 export interface CalendarProps {
   events: EventSourceInput[];
+  selectedChannel: number;
 }
 
 export class Calendar extends React.Component<CalendarProps, {}> {
   render(): JSX.Element {
+    let eventsToDisplay: EventSourceInput[] = this.props.events;
+    if (this.props.selectedChannel != null) {
+      eventsToDisplay = [this.props.events[this.props.selectedChannel]];
+    }
```

Comment:
Nice! This could use a ternary operator to be cleaner and make `eventToDisplay` a const.

Reply:
I used it here, and also changed a function in `ChannelTable` to use the ternary operator too.